src/js/components/TextInput/TextInput.js

```diff
@@ -260,13 +270,15 @@ const TextInput = forwardRef(
       // a suggestion button or the scrollbar, give it back
       inputRef.current.focus();
       inputRef.current.value = suggestion; // needed for uncontrolled cases
-      closeDrop();
+
       if (handleSuggestionSelect) {
         if (event.persist) event.persist();
         const adjustedEvent = event;
         adjustedEvent.suggestion = suggestion;
         handleSuggestionSelect(adjustedEvent);
+        setSuggestionsAtClose();
```

Comment:
Why is `setSuggestionsAtClose()` being called here? And why is suggestionsAtClose being set to an empty value?